commands/command_migrate.go

```diff
@@ -31,6 +31,11 @@ var (
 	// of updated references.
 	migrateSkipFetch bool
 
+	// migrateImportAboveFmt indicates the presence of the --amount=<size>
```

Comment:
One very minor thing I'm just noticing now is that this comment should probably say `--above=<size>` instead of `--amount=<size>`, but I won't bother to make approval conditional on this change as it's not too important, just a nice-to-have fixup.

Reply:
done